spec/features/correct_school_spec.rb

```diff
@@ -0,0 +1,119 @@
+require "rails_helper"
+
+RSpec.feature "Logs in with TID, confirms teacher details and displays school from TPS" do
+  include OmniauthMockHelper
+
+  # create a school eligible for ECP and LUP so can walk the whole journey
+  let!(:policy_configuration) { create(:policy_configuration, :additional_payments) }
+  let!(:eligible_school) { create(:school, :combined_journey_eligibile_for_all) }
+  let!(:ineligible_school) { create(:school, :early_career_payments_ineligible, :levelling_up_premium_payments_ineligible) }
+  let(:trn) { "1234567" }
+
+  before do
+    freeze_time
+    set_mock_auth(trn)
+  end
+
+  after do
+    set_mock_auth(nil)
+    travel_back
+  end
+
+  scenario "Selects suggested school" do
+    navigate_to_correct_school_page(tps: :inside_window, school: eligible_school)
+
+    # - correct-school page
+    expect(page).to have_text(eligible_school.name)
+    expect(page).not_to have_text("Enter the school name or postcode. Use at least three characters.")
+
+    # - Select the suggested school
+    choose(eligible_school.name)
+    click_on "Continue"
+
+    expect(page).to have_text(I18n.t("early_career_payments.questions.nqt_in_academic_year_after_itt.heading"))
+
+    Claim.order(created_at: :desc).limit(2).each do |c|
```

Comment:
Not so much a PR review question, but a gap in my knowledge. Why do we expect 2 claims here?

Reply:
Let's chat about that.

Reply:
@mark-pkgflow 

https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/0a884dcfaa920e7ff28f669ada0381ba10c71165/app/controllers/concerns/part_of_claim_journey.rb#L57-L68